api/v1alpha1/database_webhook.go

```diff
@@ -58,41 +68,46 @@ func instanceSpecDefaulterForCreate(r *Database) {
 
 	// time machine defaulter logic
 
-	tmInfo := r.Spec.Instance.TMInfo
-	if tmInfo.Name == "" {
-		tmInfo.Name = r.Spec.Instance.DatabaseInstanceName + "_TM"
+	// initialize TM block if it's not provided by the user
+	if r.Spec.Instance.TMInfo == nil {
+		databaselog.Info("Initialzing to empty...", "tmInfo", r.Spec.Instance.TMInfo)
```

Comment:
Can we just log a single statement like Initializing TMInfo with default values? 
The current log would be something like this: `Initializing to empty... {CONTEXT DATA, "tmInfo":"{}"}`, this will not be very helpful in understanding what is being initialized with default values. Also, the empty value `{}` doesn't add any value to the log, what do you think?

Reply:
Makes sense, I can simply log "Initialzing empty Profiles (or TMInfo)". Not using the "default" word since we are yet to populate the default values in the struct at that point of time. We will see only zero-values for the attributes at that point.